src/routes/login/components/LoginPage.js

```diff
@@ -29,6 +29,7 @@ module.exports = function SignupPage({ onPressLogin, isAuthenticated }) {
         setIsSubmitting(false);
         if (!err) {
             // Login and redirect
+            window.location.href = '/exclusive';
```

Comment:
I think eventually we'll want to use the `useNavigate()` hook in `react-router-dom`, but I don't _think_ we're using a recent enough version of that yet (I haven't looked into it too hard, besides getting a `useNavigate is not a function` error)

Reply:
Ah, so if memory serves, `useNavigate` is from `react-navigation`, not `react-router-dom` , i think?

Would it work to dispatch an action to the router slice of the middle-end? I'm not actually sure the goal here, so probably muddying the waters; what's the `/exclusive` route?

Reply:
`/exclusive` is just my questionable name for an authenticated route, only available to users who have signed in. It was a name I felt clever about two years ago for https://github.com/BigRoomStudios/strangeluv/pull/242 and have maintained here for consistency, but I'm very, very open to changing this to better suggestions (`/protected`, `/authenticated`, etc).

`react-router-dom` has a [useNavigate hook](https://reactrouter.com/en/main/hooks/use-navigate) while `react-navigation` has [useNavigation](https://reactnavigation.org/docs/use-navigation/), but I think we're still using `strange-router` for `strangeluv` in lieu of `react-navigation`, so these waters are already pretty swampy, if you catch my drift.

Reply:
`redirect` seems like the way to go @bmleight, but we haven't upgraded to `react-router-dom` 6.x yet (it's kind of tied up with the React 18 upgrade we'll get to eventually), so we don't have access to it yet.

Reply:
gah!  I thought I looked, but probably just saw the `Redirect` component and didn't put it all together.  I'd probably opt for [dispatching a push](https://github.com/BigRoomStudios/strangeluv/blob/1b2ffd71a8d9908a23bf9ce990ee627873dd2b8b/src/middle-end/router.js#L15) from the middle end for now.  I also don't remember what was holding us up on the upgrade, but faintly remember an issue with upgrading history.  Probably time to revisit those upgrades.